news/5889.bugfix

```diff
@@ -0,0 +1 @@
+Fixed crash when installing a requirement from url that comes from a dependency without a URL.
```

Comment:
"from url" should be "from a URL" (missing "a" and make the casing match the other "URL").